fdbserver/DataDistributionTracker.actor.cpp

```diff
@@ -36,12 +36,13 @@ enum BandwidthStatus {
 struct ShardMetrics {
 	StorageMetrics metrics;
 	double lastLowBandwidthStartTime;
+	int shardCount;
```

Comment:
Does `shardCount` need to be reset to 1 after merging?

Reply:
It will get reset to 1 when trackShardBytes detects that the merge has actually been carried out by the queue